misc/vim/ftdetect/factor.vim

```diff
@@ -1 +1,3 @@
-autocmd BufRead,BufNewFile *.factor,{,.}factor*-rc set filetype=factor
+" autocmd BufRead,BufNewFile *.factor setf factor " upstreamed
```

Comment:
why is this commented out?

Reply:
https://github.com/vim/vim/commit/520470a9d6afa273294848bc31e9ddfabe130434#diff-44a099e40ae628aef90a28c6408a4c61R546-R548

https://github.com/vim/vim/blob/975a880a1389e8ce6dea8d66a7c109140b2f94ec/runtime/filetype.vim#L583-L584

I figured to leave it in though for demonstrating where you'd handle new (non-`factor-doc`) Factor file extensions, if necessary.